2996_reflection/reflection.md

```diff
@@ -512,13 +520,13 @@ constexpr std::string enum_to_string(E value) {
 
 Note that this last version has lower complexity: While the versions using an expansion statement use an expected O(N) number of comparisons to find the matching entry, a `std::map` achieves the same with O(log(N)) complexity (where N is the number of enumerator constants).
 
-On Compiler Explorer: [EDG](https://godbolt.org/z/Y5va8MqzG), [Clang](https://godbolt.org/z/Kfqc77rMq).
+On Compiler Explorer: [EDG](https://godbolt.org/z/Y5va8MqzG), [Clang](https://godbolt.org/z/3doherKx8).
 
 
 Many many variations of these functions are possible and beneficial depending on the needs of the client code.
 For example:
 
-  - the "<unnamed>" case could instead output a valid cast expression like "E(5)"
+  - the "\<unnamed>" case could instead output a valid cast expression like "E(5)"
```

Comment:
typo, stray slash?
```suggestion
- the "<unnamed>" case could instead output a valid cast expression like "E(5)"
```

Reply:
Remarkably _not_ a typo: This otherwise renders as `""` because it looks like an HTML tag.

Reply:
Oh... snap.